packages/cli-v3/src/commands/deploy.ts

```diff
@@ -330,6 +331,7 @@ async function _deployCommand(dir: string, options: DeployCommandOptions) {
         loadImage: options.loadImage,
         buildPlatform: options.buildPlatform,
         noCache: options.noCache,
+        extraCACerts: resolvedConfig.config.extraCACerts?.replace(/^\./,"/app") ?? "",
```

Comment:
ideally it should be used with `path.resolve`. Should I move this into `resolveConfig`? I am thinking of copying this value into `additionalFiles` and convert this value to absolute path by `resolve('/app",config.additionalCACerts)`

Reply:
Agreed on `path.resolve()` and on moving everything to `resolveConfig()` :+1: 

Would be great if `extraCACerts` could accept paths outside the project dir, just like `additionalFiles` does: https://github.com/triggerdotdev/trigger.dev/blob/f91a118958fdfcbc6a3f8bafe8c9867db7e26640/packages/cli-v3/src/commands/deploy.ts#L1878-L1884

Essentially, just strip all `..` segments. The posix path sep is due to the requirement of glob patterns. (Yes, there are potential issues with this, but it's what additionalFiles currently does.)

`additionalFiles` will handle the raw paths fine, but we'll have to transform the `resolvedConfig.config.extraCACerts` output.

Reply:
I was unable to test relative path scenario, I say I was unable to set config correctly, if set correctly, the new regex would replace any instance of `./` or `../../../` to empty string making it relative to `/app` directory.